courses/machine_learning/images/mnist_models.ipynb

```diff
@@ -0,0 +1,1012 @@
+{
```

Comment:
<p>Running the cell above  gave me the following error (I am running the notebook locally):</p><p> </p><p>2019-10-14 17:15:59.676363: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA</p><p>2019-10-14 17:15:59.701014: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x1327e9970 executing computations on platform Host. Devices:</p><p>2019-10-14 17:15:59.701035: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): Host, Default Version</p><p>Traceback (most recent call last):</p><p>  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 193, in _run_module_as_main</p><p>    "__main__", mod_spec)</p><p>  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 85, in _run_code</p><p>    exec(code, run_globals)</p><p>  File "/Users/dherin/Desktop/asl/training-data-analyst/courses/machine_learning/images/mnist_trainer/trainer/task.py", line 46, in &lt;module&gt;</p><p>    main()</p><p>  File "/Users/dherin/Desktop/asl/training-data-analyst/courses/machine_learning/images/mnist_trainer/trainer/task.py", line 42, in main</p><p>    model_layers, args.epochs, args.steps_per_epoch, args.job_dir)</p><p>  File "/Users/dherin/Desktop/asl/training-data-analyst/courses/machine_learning/images/mnist_trainer/trainer/model.py", line 94, in create_and_train_model</p><p>    callbacks=callbacks)</p><p>  File "/Users/dherin/Desktop/asl/venv/lib/python3.6/site-packages/tensorflow_core/python/keras/engine/training.py", line 728, in fit</p><p>    use_multiprocessing=use_multiprocessing)</p><p>  File "/Users/dherin/Desktop/asl/venv/lib/python3.6/site-packages/tensorflow_core/python/keras/engine/training_distributed.py", line 789, in fit</p><p>    *args, **kwargs)</p><p>  File "/Users/dherin/Desktop/asl/venv/lib/python3.6/site-packages/tensorflow_core/python/keras/engine/training_distributed.py", line 776, in wrapper</p><p>    mode=dc.CoordinatorMode.INDEPENDENT_WORKER)</p><p>  File "/Users/dherin/Desktop/asl/venv/lib/python3.6/site-packages/tensorflow_core/python/distribute/distribute_coordinator.py", line 782, in run_distribute_coordinator</p><p>    rpc_layer)</p><p>  File "/Users/dherin/Desktop/asl/venv/lib/python3.6/site-packages/tensorflow_core/python/distribute/distribute_coordinator.py", line 344, in _run_single_worker</p><p>    assert strategy</p><p>AssertionError</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Hey Benoit, I've come to the conclusion that it's probably best for AI-Platform to officially support TF2 rather than go down a rabbit hole trying to fix this. Is that ok with you? I took your suggestion and removed it for now and added a reminder to your review sheets to add it back in when AI-Platform is ready.</p>